pkg/hubble/parser/threefour/parser.go

```diff
@@ -76,6 +79,13 @@ func New(
 	// encounters a layer it doesn't have a parser for, instead of returning
 	// an UnsupportedLayerType error.
 	packet.decLayer.IgnoreUnsupported = true
+	args := &options.Options{
+		EnableL3L4PolicyEnrichment: true,
+	}
+
+	for _, opt := range opts {
```

Comment:
since there are existing options (the ones obtained from the parserOptions.Redact ) I'd check if this loop isn't already present in some nested call here (those optins must have been getting execute somewhere, unless i read it wrong)

Reply:
Each new Parser is 'newed' up with the options and they are used to update the args default values. I think this is ok here.